src/js/slider.js

```diff
@@ -0,0 +1,101 @@
+const SLIDETIME = 600;
+
+const leftBtn = document.querySelector(".jsSlideLeft");
+const rightBtn = document.querySelector(".jsSlideRight");
+const allItems = [...document.querySelectorAll(".jsSlide")];
+let clickable = true;
+let active = null;
+let newActive = null;
+
+function initSlider() {
```

Comment:
to understand the problem of that module, try to add the same slider right below the current one on the page.